impeller/renderer/backend/gles/proc_table_gles.cc

```diff
@@ -346,6 +348,8 @@ static bool ResourceIsLive(const ProcTableGLES& gl,
       return gl.IsRenderbuffer(name);
     case DebugResourceType::kFrameBuffer:
       return gl.IsFramebuffer(name);
+    case DebugResourceType::kFence:
+      return true;
```

Comment:
GLES3 has a `glIsSync` API that can be used here

Reply:
This is only used for debug labeling which I don't believe can be done to the sync fence APIs